build.sh

```diff
@@ -13,6 +13,10 @@ export FC=${FC:-mpif90}
 BUILD_DIR=${BUILD_DIR:-${UFS_MODEL_DIR}/build}
 mkdir -p "${BUILD_DIR}"
 
+if [[ "$(uname -s)" == "Darwin" ]]; then
+   gsed -i'.backup' "s:LINKER_LANGUAGE Fortran:LINKER_LANGUAGE CXX:" ./CMakeLists.txt
+fi
+
```

Comment:
This change should be made in CMakeLists.txt not here.

Reply:
Oh, OK, thank you! Will make a change in CMakeLists.txt instead

Reply:
Made the change, so the build.sh script remains the same, but CMakeLists.txt is modified for the macosx.gnu platform.